### PR TITLE
CHANGE: to postgresql

### LINKED ISSUE #211: Swtich from MariaDB to Postgres

### PR DESCRIPTION
**Pull request by iTitus**
Closes #211

### CODE REVIEW

File: doc/documentation/04_solution_strategy.adoc [hunk 1]
```diff
@@ -27,15 +27,14 @@ Nun kann man die Annotation `@WithMockKeycloakAuth` benutzen, um Tests mit einer
 === Datenbank
 
 ****
-Zur Auswahl standen hier MySQL, MariaDB und PostgreSQL, da wir damit bereits Erfahrungen gesammelt haben.
-Wir haben uns nach einer kurzen Diskussion für MariaDB entschieden.
-Und zwar weil MariaDB die aktive Open-Source-Weiterentwicklung von MySQL ist, womit wir am meisten Erfahrung haben.
+Zur Auswahl standen hier MySQL, MariaDB und PostgreSQL, da wir mit diesen Datenbanken bereits Erfahrungen gesammelt haben.
+Wir haben uns nach einer kurzen Diskussion für PostgreSQL entschieden.
+Der Großteil unserer Erfahrungen beruhte bisher zwar auf MySQL, aber wir haben uns trotzdem für PostgreSQL entschieden,
+weil es von den Server-Admins der HHU bevorzugt wird..
```
Comment (Segelzwerg) [inline doc/documentation/04_solution_strategy.adoc]:
```suggestion
Wir haben uns nach einer kurzen Diskussion für PostgreSQL entschieden.
Der Großteil unserer Erfahrungen beruhte bisher zwar auf MySQL dann haben wir uns zunächst für die open source Variante MariaDB entschieden. Nach einiger Entwicklung sind wir dann doch auf PostgreSQL umgestiegen,
weil es von den Server-Admins der HHU bevorzugt wird und für ein Testdeployment auf heroku notwendig war.
```

**Comment by Segelzwerg** [comment]
@iTitus thx for the help: approved

File: docker-compose.dev.yml [hunk 2]
```diff
@@ -4,12 +4,11 @@ services:
   minio_dev:
     image: minio/minio:latest
     ports:
-      - 9000:9000
+      - "${MINIO_PORT:-9000}:9000"
```
Comment (realemx) [inline docker-compose.dev.yml]: THis Container is only for development and the Port is given as default 9000 in the application properties. No need to change it by env variable.

File: docker-compose.dev.yml [hunk 3]
```diff
@@ -4,12 +4,11 @@ services:
...
     volumes:
       - minio_dev_volume:/data
-    environment:
-      MINIO_ACCESS_KEY: dev_minio_access_key
-      MINIO_SECRET_KEY: dev_minio_secret_key
+    env_file:
+      - prod.env
```
Comment (realemx) [inline docker-compose.dev.yml]: See above. This container is not for production.

File: docker-compose.dev.yml (hunk 2 above)
Comment (iTitus) [reply docker-compose.dev.yml]: I basically used the `prod-template.env` to list all our environment variables and the templates. The name is a bit misleading. You got any suggestions?

Comment (realemx) [reply docker-compose.dev.yml]: Maybe an extra dev.env? We could then remove the defaults in theapplication properties and make it required and less prone for error caused by different values?

**Pull request merged**